src/player.js

```diff
@@ -2601,6 +2605,26 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Sets the video track selected by the user.
+   * @returns {void}
+   * @private
+   */
+  _setDefaultVideoTrack() {
+    const sortedVideoTracks = this._getVideoTracks().sort((track1: VideoTrack, track2: VideoTrack) => track2.bandwidth - track1.bandwidth);
+    let selectedVideoTrack = sortedVideoTracks.find(
+      (track: VideoTrack) => track.label && track.label === this._playbackAttributesState.videoTrack?.label
```

Comment:
`track.label &&` is not redundant ?, anyway you can use the Optional chaining operator like you did later

Reply:
to ignore the case (mostly in tests) when the label/height is undefined so it equal `this._playbackAttributesState.videoTrack?.label`